### ISSUE TITLE
Unable to query dispatch info

### ISSUE THREAD

**Issue by transxask** [paritytech/subxt#195]
in the lastest version substrate, send extrinsics by subxt and the error occur:
```
Error: 1: Unable to query dispatch info.: Execution, Wasmi, Trap, Trap { kind: Unreachable 
```

**Comment by ascjones** [comment]
If it's substrate master it may not work, this is currently targeting the released version `2.0.0`

**Comment by ascjones** [comment]
Let me know if this is still an issue with the latest `master`
